### DISCUSSION TITLE
Using ya-webadb and Android Studio at the same time.

### DISCUSSION THREAD

**Discussion by kainbryanjones** [yume-chan/ya-webadb#550] [Q&A]
I see on Tango that you said 'One USB device can only be accessed by one application at a time'.
For the app we're building using Android Studio simultaneously will be a common use case.

Is there any way around this and/or plans to implement this in the future?

**Comment by yume-chan** [comment]
See #549.

Browsers doesn't support TCP connection, so it won't work, unless having some extra proxy software running on computer.

You can also try https://github.com/DeviceFarmer/adbkit which (only) supports connecting to ADB servers.
